lib/runalyze.lib.options.js

```diff
@@ -43,6 +43,16 @@ Runalyze.Options = (function($, parent){
 		return self;
 	};
 
+	self.setMapboxLayerAuth = function(accesstoken) {
+		options.MapboxApikey = accesstoken;
```

Comment:
This options key needs to be defined in https://github.com/chpusch/Runalyze/blob/3c66ed1b53e4fee87f91e345461d3df538f17344/lib/runalyze.lib.options.js#L20. Otherwise `MapboxAuth()` will throw an error if `setMapboxLayerAuth(...)` has not been called.